public/app.js

```diff
@@ -0,0 +1,57 @@
+// GLOBAL APP CONTROLLER
+class Controller {
+
+    static setupEventListeners() {
+        let DOM = UIController.getDOMstrings();
+        document.querySelector(DOM.logoutBtn).addEventListener('click', Controller.ctrlLogoutDashboard);
+        document.querySelector(DOM.codingProgressLink).addEventListener('click', Controller.ctrlDisplayCodingProgress);
+        document.querySelector(DOM.projectMenu).addEventListener('click', Controller.ctrlDisplayProject);          
+    };
+    
+    static ctrlLogoutDashboard() {
+        AuthController.logout()
+    };
+
+    // Navigate to coding progress page
+    static ctrlDisplayCodingProgress(e) {
```

Comment:
```suggestion
    static navigateToCodingProgress(e) {
```
Then you don't need a comment explaining your function name :)